### PR TITLE
#159236744 Setup sass pre-processing

### PR DESCRIPTION
**Pull request by d3mola**
#### What does this PR do?
This PR sets up `sass` pre-processing to allow the use of `.scss` and `.css` files

#### Description of Task to be completed?
- install node-sass and sass-loader as dependencies
- use loaders in the webpack config files to convert .scss to .css files
- change all instances of .css to use .scss in App.jsx and index.
- rename index.css and App.css to index.scss and App.scss repectively

#### How should this be manually tested?
- Clone the repo `git clone https://github.com/andela/travel_tool_front.git`
- Change into the folder `cd travel_tool_front`
- Switch to this branch `git checkout ch-setup-sass-159236744`
- Install dependencies `yarn install`
- Start the app and run in a browser `yarn start`
- Edit the `scss` files or create a new one

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#159236744](https://www.pivotaltracker.com/story/show/159236744)

#### Screenshots (if appropriate)
screen shot 2018-07-24 at 9 43 30 pm

#### Questions:
N/A

### CODE REVIEW

**Comment by samuelkubai** [comment]
@d3mola for the config file copy the previous configuration for the .css and add the sass-loader after style-loader to preserve postcss capability that's already configured for the css files.

Here is a resource to guide you.
https://medium.com/front-end-hacking/how-to-add-sass-or-scss-to-create-react-app-c303dae4b5bc

File: config/webpack.config.dev.js [hunk 1]
```diff
@@ -188,6 +188,15 @@ module.exports = {
...
           },
+          {
+            test: /\.scss$/,
+            include: paths.appSrc,
+            loaders: [
+              require.resolve('style-loader'),
+              require.resolve('css-loader'),
+              require.resolve('sass-loader')
+            ],
+          },
```
Comment (samuelkubai) [inline config/webpack.config.dev.js]: This is the part that I am talking about.

File: config/webpack.config.prod.js [hunk 2]
```diff
@@ -212,6 +212,15 @@ module.exports = {
...
           },
+          {
+            test: /\.scss$/,
+            include: paths.appSrc,
+            loaders: [
+              require.resolve('style-loader'),
+              require.resolve('css-loader'),
+              require.resolve('sass-loader')
+            ],
+          },
```
Comment (samuelkubai) [inline config/webpack.config.prod.js]: This part too. Production configs have more to configure as you will be guided by the resource I shared.

Comment (d3mola) [reply config/webpack.config.prod.js]: Thanks for the resource. I'll check it out and make necessary changes.

**Comment by samuelkubai** [comment]
@d3mola please update your branch from develop and resolve the conflicts.

**Pull request merged**